apps/trading-e2e/src/integration/market-info.cy.ts

```diff
@@ -148,6 +160,22 @@ describe('market info is displayed', () => {
       'Termination Oracle',
       'f028fe5ea7de3890962a05a7163fdde562629af649ed81b8c8902fafb6eef04f'
     );
+
+    cy.getByTestId(externalLink)
+      .should('have.attr', 'href')
+      .and('contain', 'https://explorer.fairground.wtf/oracles');
```

Comment:
Question (non-blocking): Should these urls be environment-aware? I'm wondering if we'd ever want to run these tests against mainnet

Reply:
That's a valid point. I've loosened the assertion so it should still pass when if we ever want to run these tests on another environment.